### PR TITLE
TELCODOCS-1480 - Adding new and missing PTP events

### PR DESCRIPTION
**Pull request by aireilly**
Adding new and missing PTP events for https://issues.redhat.com//browse/TELCODOCS-1480 - PTP T-GM dual NIC config for D-RAN use cases.

Version(s):
enterprise-4.14+

Issue:
https://issues.redhat.com/browse/TELCODOCS-1480

Link to docs preview:
https://70235--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ptp/using-ptp-events#nw-ptp-operator-metrics-reference_using-ptp-hardware-fast-events-framework

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Review by jzding — APPROVED** [review]
/lgtm

File: modules/nw-ptp-operator-metrics-reference.adoc [hunk 1]
```diff
@@ -22,54 +22,78 @@ Some of the following metrics are applicable for PTP grandmaster clocks (T-GM) o
...
-|`openshift_ptp_frequency_adjustment_ns {from="phc", iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} -6768`
+|`{from="phc", iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} -6768`
+
+|`openshift_ptp_frequency_status`
+|Returns the current status of the DPLL frequency for the NIC.
+Possible values are -1 (`UNKNOWN`), 0 (`INVALID`), 1 (`FREERUN`), 2 (`LOCKED`), 3 (`LOCKED_HO_ACQ`), or 4 (`HOLDOVER`).
+|`{from="dpll",iface="ens2fx",node="compute-1.example.com",process="dpll"} 3`
+
+|`openshift_ptp_phase_status`
+|Returns the status of the DPLL phase for the NIC.
```
Comment (jab-rh) [inline modules/nw-ptp-operator-metrics-reference.adoc:50]: Does it make sense to mention this is _Digital Phase-Locked Loop_ at first usage here?

File: modules/nw-ptp-operator-metrics-reference.adoc [hunk 2]
```diff
@@ -22,54 +22,78 @@ Some of the following metrics are applicable for PTP grandmaster clocks (T-GM) o
...
 |`openshift_ptp_threshold`
 a|Returns values for `HoldOverTimeout`, `MaxOffsetThreshold`, and `MinOffsetThreshold`.
 
 * `holdOverTimeout` is the time value in seconds before the PTP clock event state changes to `FREERUN` when the PTP master clock is disconnected.
 * `maxOffsetThreshold` and `minOffsetThreshold` are offset values in nanoseconds that compare against the values for `CLOCK_REALTIME` (`phc2sys`) or master offset (`ptp4l`) values that you configure in the `PtpConfig` CR for the NIC.
-|`openshift_ptp_threshold {node="compute-1.example.com", profile="grandmaster", threshold="HoldOverTimeout"} 5`
+|`{node="compute-1.example.com", profile="grandmaster", threshold="HoldOverTimeout"} 5`
+
+|`openshift_ptp_pps_status`
+|Returns the current status of the NIC PPS connection.
```
Comment (jab-rh) [inline modules/nw-ptp-operator-metrics-reference.adoc:86]: What's PPS, NMEA, or GNSS?

**Review by jab-rh — COMMENTED** [review]
@aireilly, minor comments, thanks!

**Review by jzding — APPROVED** [review]
/lgtm

**Comment by aireilly** [comment]
@jzding ^^ for the clock class updates for https://github.com/openshift/linuxptp-daemon/pull/266

**Comment by aireilly** [comment]
NOTE: This is not merged to 4.15 yet - waiting for the feature to be released on 4.15.

**Pull request merged**